src/utils/checkUpdateAvailable.ts

```diff
@@ -1,23 +1,35 @@
 import chalk from 'chalk';
 import semver from 'semver';
-import fetch from 'node-fetch';
+import fetch, { Response } from 'node-fetch';
+
 // eslint-disable-next-line @typescript-eslint/no-var-requires
-const { version, name } = require('../../package.json');
+const { version: localVersion, name } = require('../../package.json');
+
+const logAvailableUpdate: (upstreamVersion: string) => void = (
+  upstreamVersion: string,
+): void => {
+  console.log(
+    `${name} update available from ${chalk.greenBright(
+      localVersion,
+    )} to ${chalk.greenBright(upstreamVersion)}.`,
+  );
+};
 
 export default async (): Promise<void> => {
+  let res: Response;
+
   try {
-    const res = await fetch(
+    res = await fetch(
       'https://api.github.com/repos/bettyblocks/cli/releases/latest',
     );
-    const data = await res.json();
-    const upstreamVersion = data.tag_name.substring(1);
+  } catch {
+    return;
+  }
+
+  const { tag_name: tagName } = await res.json();
```

Comment:
Can we add:
```
  if (!tagName) {
    return;
  }
```

Otherwise we'll get an tagName is undefined when the request return s anything else than the success response.

Reply:
Added